usaspending_api/common/helpers/generic_helper.py

```diff
@@ -188,7 +188,6 @@ def within_one_year(d1, d2):
             days_diff -= 1
     return days_diff <= 365
 
-
```

Comment:
This is such a terribly petty thing and I feel absolutely awful for adding this comment, but can we put this blank line back?  My linter is complaining that it wants two spaces between functions and, honestly, this seems like an unnecessary change.